ozaria/site/views/play/level/tome/SpellView.coffee

```diff
@@ -1141,7 +1141,7 @@ module.exports = class SpellView extends CocoView
   focus: ->
     # TODO: it's a hack checking if a modal is visible; the events should be removed somehow
     # but this view is not part of the normal subview destroying because of how it's swapped
-    return unless @controlsEnabled and @writable and $('.modal:visible, .shepherd-button:visible').length is 0
+    return unless @controlsEnabled and @writable and $('.modal:visible').length is 0
```

Comment:
I just added this a few weeks ago in https://github.com/codecombat/codecombat/commit/2e12e3b925021bfb72bebd46ec58dd7a5951f3e0. Without it, keyboard navigation of shepherd buttons doesn't really work except in screen reader mode, and it messes up the tab order.

Does the new implementation still let us use keyboard navigation on shepherd buttons when screen reader mode isn't activated?

Reply:
@nwinter 
Because the focus will be in the editor, the keyboard arrows will move the cursor instead of navigating shepherd.
I've checked if the keyboard navigation works on production, but I found it doesn't work as I would expect:
 ![navigation](https://user-images.githubusercontent.com/11805970/186632225-f393493f-7c3d-4d4b-97e1-7a08a3d6d281.gif)

Reply:
I tested this again and it seems to behave the same both with and without this change (it mostly works but you still need to click into the Shepherd modal to get keyboard focus to target it sometimes). It's possible I'm testing wrongly or have forgotten what my original edge case was. Anyway, let's go with your fix and improve further as needed in the future.